libraries/vlfeat/CMakeLists.txt

```
@@ -1,6 +1,6 @@
 project(vlfeat)
 
-include("${CMAKE_MODULE_PATH}/OptimizeTheiaCompilerFlags.cmake")
+include("${CMAKE_SOURCE_DIR}/cmake/OptimizeTheiaCompilerFlags.cmake")
```

```suggestion
include(OptimizeTheiaCompilerFlags)
```
`CMAKE_MODULE_PATH` was modified to find that module. `CMAKE_SOURCE_DIR` is wrong if theia is included via add_subdirectory by a parent project.